src/Tizen.Multimedia.MediaPlayer/Player/Player.cs

```diff
@@ -549,15 +551,17 @@ public int GetPlayPosition()
             return playPosition;
         }
 
-        private void SetPlayPosition(int milliseconds, bool accurate,
+        private void SetPlayPosition(long position, bool accurate, bool nanoseconds,
             NativePlayer.SeekCompletedCallback cb)
         {
-            var ret = NativePlayer.SetPlayPosition(Handle, milliseconds, accurate, cb, IntPtr.Zero);
+            //Check if it is nanoseconds or milliseconds.
+            var ret = !nanoseconds ? NativePlayer.SetPlayPosition(Handle, Convert.ToInt32(position), accurate, cb, IntPtr.Zero) :
```

Comment:
Does ```Convert.ToInt32(position)``` really needed? 
Isn't it enough just explicit casting ```(int)position```?

Reply:
i think it is not needed. i'll change it to cast only.